transports/astartehttpendpoint.cpp

```diff
@@ -201,7 +200,9 @@ void HTTPEndpoint::initImpl()
 
     QSettings settings(d->configurationFile, QSettings::IniFormat);
     settings.beginGroup(QStringLiteral("AstarteTransport")); {
-        if (settings.contains(QStringLiteral("pairingJwt"))) {
+        if (settings.contains(QStringLiteral("credentialsSecret"))) {
+            d->credentialsSecret = settings.value(QStringLiteral("credentialsSecret")).toString().toLatin1();
+        } else if (settings.contains(QStringLiteral("pairingJwt"))) {
```

Comment:
I'd detect if both `credentialsSecret` and `pairingJwt` are set and print a warning